views/index.pug

```diff
@@ -1,9 +1,6 @@
 extends layout
 
 block content
-  a(href="/users/new") Sign Up
-  br
-  a(href="/session/new") Log In
-  h1= title
-  p Welcome to #{title}
-
```

Comment:
Is this going to be the landing page? Will this view be visible to people who have accounts?

Reply:
Yes it's the landing page, it should be visible to everyone